app.freelens.Freelens.yaml

```diff
@@ -0,0 +1,104 @@
+id: app.freelens.Freelens
+base: org.electronjs.Electron2.BaseApp
+base-version: "24.08"
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: freelens
+separate-locales: false
+finish-args:
+  # X11 performance
+  - --share=ipc
+  # We need X11
+  - --socket=fallback-x11
+  # Access to wayland
+  - --socket=wayland
+  # Access to all files
+  # User's files
+  - --persist=~/.freelens
+  - --filesystem=~/.kube
+  # Run commands on host
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
what commands does it need to run and why? This needs an exception.

Reply:
1. Freelens is not just UI for Kubernetes: it is also terminal application like org.kde.konsole or com.visualstudio.code. In the terminal you run your shell with your scripts from home directory.
2. Kubectl also needs a support for custom authentication. It must run any command that gets the token from the cloud provider: az, aws, gcloud, others. Basically, it might be any command from the host.

Reply:
this needs an exception https://docs.flathub.org/docs/for-app-authors/linter#exceptions, please open a PR 


code https://docs.flathub.org/docs/for-app-authors/linter#finish-args-flatpak-spawn-access